docs/user-guide/managing-deployments.md

```diff
@@ -439,6 +441,27 @@ $ rm $TMP
 
 The system ensures that you don’t clobber changes made by other users or components by confirming that the `resourceVersion` doesn’t differ from the version you edited. If you want to update regardless of other changes, remove the `resourceVersion` field when you edit the resource. However, if you do this, don’t use your original configuration file as the source since additional fields most likely were set in the live state.
 
+## Using configuration files
+
+A more disciplined alternative to patch and edit is `kubectl apply`. 
```

Comment:
@janetkuo Note that commit SHA will change when the other edits requested here are pushed.